_posts/2014-05-28-geomesa-quickstart.md

```diff
@@ -188,17 +189,19 @@ Click "Save", and GeoServer will search your Accumulo table for any GeoMesa-mana
 
 GeoServer should recognize the `QuickStart` feature type, and should present that as a layer that could be published.  Click on the "Publish" link.
 
-You will be taken to the Edit Layer screen.  Two of the tabs need to be updated:  Data and Dimensions.
+You will be taken to the Edit Layer screen. Two of the tabs need to be updated: Data and Dimensions.
 
 In the Data pane, enter values for the bounding boxes.  In this case, you can click on the link to compute these values from the data.
 
+In the Dimensions tab, check the "Enabled" checkbox. Then select "When" in the Attribute and End Attribute dropdowns, and "Continuous Interval" in the Presentation dropdown.
```

Comment:
We may also want to refer to the following URL as a discussion of when the TIME parameter in WMS queries is available in GeoServer, and what sort of formats it accepts (as we're making changes anyway):  http://docs.geoserver.org/latest/en/user/services/wms/time.html